types/reactstrap/index.d.ts

```diff
@@ -193,3 +193,5 @@ export {
   UncontrolledCarouselProps,
   UncontrolledPopoverProps
 } from './lib/Uncontrolled';
+import * as Util from './lib/Util';
```

Comment:
`export { Util } from './lib/Util':` to match the style in the file?

Reply:
reactstrap exports the functions inside Util with `export * as Util from './utils'` but it seems this is not available until Typescript 3.8. 

Would it be better to put the functions in Util under a namespace and export that?

Util.d.ts
```
declare namespace Util {
    export function setGlobalCssModule(cssModule: CSSModule): void;
}

export default Util;
```

index.d.ts
```
export { default as Util } from './lib/Util';
```

Reply:
This looks ok